examples/single_calculations/example_dft_md_reftraj.py

```diff
@@ -0,0 +1,179 @@
+                    "REFTRAJ": {
+                        "FIRST_SNAPSHOT": 1,
+                        "LAST_SNAPSHOT": steps,
+                        "EVAL_FORCES": ".TRUE.",
+                        "TRAJ_FILE_NAME": "trajectory.xyz",
+                        "CELL_FILE_NAME": "reftraj.cell",
+                        "VARIABLE_VOLUME": ".TRUE.",
+                    },
+                },
+                "PRINT": {
+                    "RESTART": {
+                        "EACH": {
+                            "MD": 1,
+                        },
+                    },
+                    "FORCES": {
+                        "EACH": {
+                            "MD": 1,
+                        },
+                    },
+                    "CELL": {
+                        "EACH": {
+                            "MD": 1,
+                        },
+                    },
+                },
+            },
+            "FORCE_EVAL": {
+                "METHOD": "Quickstep",
+                "DFT": {
+                    "BASIS_SET_FILE_NAME": "BASIS_MOLOPT",
+                    "POTENTIAL_FILE_NAME": "GTH_POTENTIALS",
+                    "QS": {
+                        "EPS_DEFAULT": 1.0e-12,
+                        "WF_INTERPOLATION": "ps",
+                        "EXTRAPOLATION_ORDER": 3,
+                    },
+                    "MGRID": {
+                        "NGRIDS": 4,
+                        "CUTOFF": 280,
+                        "REL_CUTOFF": 30,
+                    },
+                    "XC": {
+                        "XC_FUNCTIONAL": {
+                            "_": "LDA",
+                        },
+                    },
+                    "POISSON": {
+                        "PERIODIC": "none",
+                        "PSOLVER": "MT",
+                    },
+                },
+                "SUBSYS": {
+                    "KIND": [
+                        {
+                            "_": "O",
+                            "BASIS_SET": "DZVP-MOLOPT-SR-GTH",
+                            "POTENTIAL": "GTH-LDA-q6",
+                        },
+                        {
+                            "_": "H",
+                            "BASIS_SET": "DZVP-MOLOPT-SR-GTH",
+                            "POTENTIAL": "GTH-LDA-q1",
+                        },
+                    ],
+                },
+            },
+        }
+    )
+
+    # Construct process builder.
+    builder = cp2k_code.get_builder()
+    builder.structure = structure
+    builder.trajectory = trajectory
+    builder.parameters = parameters
+    builder.code = cp2k_code
+    builder.file = {
+        "basis": basis_file,
+        "pseudo": pseudo_file,
+    }
+    builder.metadata.options.resources = {
+        "num_machines": 1,
+        "num_mpiprocs_per_machine": 1,
+    }
+    builder.metadata.options.max_wallclock_seconds = 1 * 3 * 60
+
+    print("Submitted calculation...")
+    run(builder)
+
```

Comment:
Please add some quantities to compare with. It is a bit pointless to run a calculation without checking its results.

Reply:
Added a check for the total number of steps in all trajectories in [5ac5e24](https://github.com/aiidateam/aiida-cp2k/pull/207/commits/5ac5e24e90793790f67ddcf50771107493539a69)